openfisca_core/periods.py

```diff
@@ -673,6 +673,29 @@ def unit(self):
         return self[0]
 
 
+    # Reference periods
+
+    @property
+    def last_3_months(self):
+        return self.this_month.start.period('month', 3).offset(-3)
+
+    @property
+    def last_month(self):
+        return self.this_month.offset(-1)
+
+    @property
+    def n_2(self):
```

Comment:
I'm not sure there's a sensible way between changing the core `offset`/`period` API and this “just give me straight what I use 90% of the time”.

I think these are syntactic sugar over generic period manipulation functions and that they make sense mostly if hardcoded. Then again, if there was a way to isolate these helpers from `openfisca-core`, it would probably be better, but the current lack of modularity makes that kind of adjustments pretty hard.